shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1150,11 +1196,18 @@ - (instancetype)init {
 - (void)dealloc {
   [self hideTextInput];
   [_reusableInputView release];
+  [_inputHider release];
   [_autofillContext release];
-
   [super dealloc];
 }
 
+- (void)removeEnableFlutterTextInputViewAccessibilityTimer {
+  if (_enableFlutterTextInputViewAccessibilityTimer) {
+    [_enableFlutterTextInputViewAccessibilityTimer invalidate];
+    _enableFlutterTextInputViewAccessibilityTimer = nil;
```

Comment:
This is a leak, sorry i didn't see it earlier.  You need to retain this below, and release this here.